src/storage/disk_table.cc

```diff
@@ -875,5 +858,56 @@ uint64_t DiskTable::GetRecordIdxByteSize() {
     return 0;
 }
 
+int DiskTable::GetCount(uint32_t index, const std::string& pk, uint64_t& count) {
+    std::shared_ptr<IndexDef> index_def = table_index_.GetIndex(index);
```

Comment:
As disktable::GetCount is quite slow. have to iterate over the table.

log a warning message: "GetCount in disk table is slow"